src/main/scala/com/intenthq/sbt/ThriftPlugin.scala

```diff
@@ -58,77 +59,92 @@ object ThriftPlugin extends AutoPlugin {
   import autoImport._
 
   override def requires = sbt.plugins.JvmPlugin
-
   override def trigger = allRequirements
 
   val Thrift = config("thrift")
 
   val thriftSettings: Seq[Setting[_]] = inConfig(Thrift)(Seq(
     thrift := "thrift",
-    thriftSourceDir <<= sourceDirectory { _ / "main" / "thrift" },
+    thriftSourceDir := { sourceDirectory.value / "main" / "thrift" },
```

Comment:
The syntax for these sorts of things changed, see the docs here if you want more information: http://www.scala-sbt.org/1.x/docs/Migrating-from-sbt-013x.html#Migrating+sbt+0.12+style+operators